cmd/image-builder/upload.go

```diff
@@ -93,6 +93,21 @@ func uploaderForCmdAWS(cmd *cobra.Command, targetArchStr string, bootMode *platf
 	if err != nil {
 		return nil, err
 	}
+	tags, err := cmd.Flags().GetStringArray("aws-tag")
+	if err != nil {
+		return nil, err
+	}
+	var slicedTags [][2]string
+	for _, tag := range tags {
+		parts := strings.SplitN(tag, "=", 2)
+		if len(parts) != 2 {
```

Comment:
What is the policy of AWS here? Are values wit `=` allowed? Because if they are we could change this to `if len(parts) < 2` and allow `=` in the values as well.

Reply:
I am still testing this part of the PR but then I will rebase :-)